lib/config.js

```diff
@@ -22,6 +22,7 @@ const Config = function () {
   this.projects = {}
   this.signTarget = 'sign_app'
   this.buildTarget = 'brave'
+  this.baseBranch = 'master'
```

Comment:
should this be `origin/master`?

Reply:
it's very easy for local `master` to be wrong, especially after running `init` on a different branch

Reply:
also how do we override this? Should be an option when running `npm run lint`